Mathlib/Data/Real/Sqrt.lean

```diff
@@ -262,6 +264,9 @@ lemma sqrt_le_sqrt_iff' (hx : 0 < x) : √x ≤ √y ↔ x ≤ y := by
 @[simp] lemma sqrt_le_one : √x ≤ 1 ↔ x ≤ 1 := by
   rw [← sqrt_one, sqrt_le_sqrt_iff zero_le_one, sqrt_one]
 
+@[simp] lemma isSquare_iff {r : ℝ} : IsSquare r ↔ 0 ≤ r :=
```

Comment:
Let's use `x` here like the surrounding lemmas

Reply:
Done.